lib/BackbeatConsumer.js

```diff
@@ -241,6 +246,37 @@ class BackbeatConsumer extends EventEmitter {
         return this._consumer.once('ready', () => {
             this._consumerReady = true;
             this._checkIfReady();
+            if (this._logConsumerMetricsIntervalS !== undefined) {
+                this._consumer.on('event.stats', res => {
+                    const statsObj = JSON.parse(res.message);
+                    if (typeof statsObj !== 'object') {
```

Comment:
If you successfully parse a string with JSON.parse, it will always be an object. If not, the JSON.parse will throw an error. I think here we should use a safe parse function (IIRC, we have one in arsenal), or we should wrap that in a try/catch